client/css/style.css

```diff
@@ -268,6 +268,16 @@ kbd {
 #sidebar .chan.lobby::before,
 #chat .lobby .title::before { content: "\f0a0"; /* http://fontawesome.io/icon/hdd-o/ */ }
 
+#sidebar .not-secure .chan.lobby::before {
+	content: "\f09c"; /* http://fontawesome.io/icon/unlock/ */
+	color: #ff1744;
```

Comment:
Those reds are very aggressive lol. Plus it seems to me we don't use this shade of red already. Mind using `#e74c3c` with is a bit more pleasing to the eye (but still red)? We use that color in a few places, like errors in form, connection lost banner, etc.

Reply:
Actually, I have 2 suggestions on top of the previous that I think might look better:

- For insecure connection, use the `#f39c12` orange we have. After all, some networks are insecure for a good reason, not an error on the client or something the user didn't do well, so marking it as an error might be too strict (and might be confusing between the 2 states).

- Changing the color of the entire line looks nicer, more homogenous.

That would look like this:

<img width="190" alt="screen shot 2018-02-18 at 00 43 06" src="https://user-images.githubusercontent.com/113730/36348774-43e017fe-1445-11e8-9801-203c770b84be.png"><br><img width="188" alt="screen shot 2018-02-18 at 00 44 40" src="https://user-images.githubusercontent.com/113730/36348775-43ed0838-1445-11e8-97d6-7d2bc33ed770.png">

Reply:
Feel free to make a commit with all the changes.